visualize.py

```diff
@@ -50,7 +50,13 @@ def main(args=None):
 	use_gpu = True
 
 	if use_gpu:
-		retinanet = retinanet.cuda()
+        if torch.cuda.is_available():
```

Comment:
It appears you used spaces instead of tabs here

Reply:
Changed it. Somehow i don't know why it was bad indented